### ISSUE TITLE
Notice: Undefined index: jits

### ISSUE THREAD

**Issue by danaki** [LulububuSoftwareGmbH/boltredirector#14]
Comment "jits" config option and see the error in config editor.

**Comment by danaki** [comment]
Also leaving empty "jits:" config parameter gives
```
ContextErrorException in Config.php line 33:
Warning: Invalid argument supplied for foreach()
```

**Comment by danaki** [comment]
To avoid error:
```
jits: []
```

but there pops another error during redirect step:

```
ContextErrorException in Redirect.php line 114: 
Warning: Invalid argument supplied for foreach()
```

You just can't leave "jits" parameter nor empty nor undefined.

**Comment by SvanteRichter** [comment]
This should have been fixed in #20, if it is not please hit me over the head with something hard and reopen :)
